app/views/notify-token-drawer/notify-token-form.tsx

```diff
@@ -0,0 +1,71 @@
+import type { ApolloError } from '@apollo/client'
+import type { FormEventHandler } from 'react'
+import { useForm } from 'react-hook-form'
+import clsx from 'clsx'
+import styles from './wallet-linking-drawer.module.scss'
+import { FormInput } from '@/components/form-input/form-input'
+import Button from '@/components/button/button'
+
+export interface NotifyTokenFormInput {
+  token: string
+}
+
+export interface NotifyTokenFormProps {
+  onSubmit: (data: NotifyTokenFormInput) => void | Promise<void>
+  success?: boolean
+  loading?: boolean
+  error?: Error | ApolloError | null
+}
+
+export function NotifyTokenForm({
+  onSubmit,
+  loading,
+  success,
+  error,
+}: NotifyTokenFormProps) {
+  const { control, handleSubmit, formState } = useForm<NotifyTokenFormInput>()
+
+  const fullFormDisabled = loading || success
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit) as FormEventHandler<any>} className={styles.form}>
+      <div className={styles.header}>
+        <h3>What token do you use?</h3>
+      </div>
+      We working on adding more, you can share tokens that we missing and we will prioritize it.
```

Comment:
Check if it should be above or below the button?

Reply:
Better to place it below